lib/widgets/wallet_list.dart

```diff
@@ -111,14 +111,18 @@ class WalletListState extends State<WalletList> {
   }
 
   Widget _buildInitialButtons() {
+    final theme = Theme.of(context);
     return PaddedButton(
       padding: EdgeInsets.all(0),
       text: localization.createOrImportLabel,
+      color: theme.textTheme.displayMedium!.color,
+      boldText: true,
```

Comment:
The bold new fontFamily is too bold for this button. Can we change it to a lighter weight?
![Screenshot 2024-10-07 at 16 11 12](https://github.com/user-attachments/assets/0c865343-fed1-40d1-bda8-f01d89a15bde)